src/client/dfuse/il/int_posix.c

```diff
@@ -280,6 +291,26 @@ ioil_init(void)
 	ioil_iog.iog_initialized = true;
 }
 
+static void
+ioil_show_summary()
+{
+	char *summary;
+
+	D_INFO("Performed %"PRIu64" reads and %"PRIu64" writes from %"PRIu64" files\n",
+	       ioil_iog.iog_read_count, ioil_iog.iog_write_count, ioil_iog.iog_file_count);
+
+	if (ioil_iog.iog_file_count == 0)
+		return;
+
+	summary = getenv("D_IL_REPORT");
```

Comment:
This can be done in a follow-up doc-only patch, but we *really* need to document that in the manual.